synfig-studio/src/gui/instance.cpp

```diff
@@ -974,7 +974,7 @@ Instance::safe_revert()
 	{
 		if (!App::dialog_message_2b(
 			_("Revert to saved"),
-			_("You will lose any changes you have made since your last save."
+			_("You will lose any changes you have made since your last save. "
```

Comment:
Is extra space needed?

Reply:
It's because "are you share" ;)
Maybe a newline `\n` would be better.

Reply:
Without the space here is how they look:
![no_space](https://user-images.githubusercontent.com/6705690/83050465-e51c0980-a04c-11ea-992a-2cbfc3ab2124.png)

Reply:
New line as suggested by @rodolforg would be better, but that is ok too.